test/publicProfile.matchers.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { _, definePattern } = require('./matchers');
+
+definePattern(
+  'aPublicProfile',
+  {
+    avatar: _.isAssetUrl,
+    banner: _.isUndefinedOr(_.isAssetUrl),
+    bio: _.isUndefinedOr(_.isString),
+    city: _.isUndefinedOr(_.isString),
+    country: _.isUndefinedOr(_.isString),
+    did: _.isDID,
+    displayName: _.isString,
```

Comment:
we also have some rules about this text that could be tested here

Reply:
removed this whole matcher since I didn't sue it